packages/hash/api/src/model/knowledge/hashInstance.model.ts

```diff
@@ -234,4 +242,16 @@ export default class extends EntityModel {
 
     await outgoingAdminLinkEntityModel.archive(graphApi, { actorId });
   }
+
+  isUserRegistrationEnabled(): boolean {
+    return (this.getProperties() as any)[
```

Comment:
Not a suggestion for this PR, but when cleaining up the model classes I would consider having `PropertyObject` to take a generic which can then be fed through from classes that extend `Entity` to properly type their `properties`.

Reply:
I like the idea of properly typing them, but you can use `PropertyObject` here to type this a lot better (a `BaseUri` keyed JSON object)

Reply:
`getProperties()` actually now returns `PropertyObject`, so the typing can be improved slightly:
https://github.com/hashintel/hash/blob/9facccde72bf80819227de06ea2f2c02b5e3bb75/packages/hash/api/src/model/knowledge/hashInstance.model.ts#L246-L256